mlflow/server/js/src/components/ExperimentViewUtil.js

```diff
@@ -402,10 +405,28 @@ export default class ExperimentViewUtil {
     return mergedRows;
   }
 
+  /**
+   * Given an array of row metadata returned by getRowRenderMetadata, returns an array of run IDs
+   * sorted by display order.
```

Comment:
I would mention that the main use-case here is to have a list of runs, including both collapsed and expanded runs.